lib/src/over_react_test/common_component_util.dart

```diff
@@ -132,14 +131,14 @@ Iterable _flatten(Iterable iterable) =>
 ///       });
 ///     }
 void expectCleanTestSurfaceAtEnd() {
-  Set<Element> nodesBefore;
+  late Set<Element> nodesBefore;
 
   setUpAll(() {
-    nodesBefore = document.body.children.toSet();
+    nodesBefore = document.body!.children.toSet();
```

Comment:
i understand that sure a document might not have a body but given how often that is a case this feels ridiculous lol

Reply:
lol true